### PR TITLE
[RFXCOM] Updated the readme

### PR DESCRIPTION
**Pull request by martinvw**
To clarify the documentation, a lot of information was added on setting up the RFXCOM binding manually.

### CODE REVIEW

**Comment by martinvw** [comment]
@kaikreuzer is our account broken again? Mention bot seems gone.

**Comment by kaikreuzer** [comment]
I can see that the webhook has been delivered - so it must be an issue on their end; hopefully a temporary one.

**Comment by martinvw** [comment]
@mjagdis could you also update the README in this branch to include all the recent changes, i.e. added/changed configuration, message types and channels. I will grant you access.

**Comment by martinvw** [comment]
@metbril I also added thing type specific information now only for blinds1 and lighting4, do you think this is the way to go?

**Comment by metbril** [comment]
@martinvw Not sure [why](#issuecomment-306039049) you ask me.

**Comment by metbril** [comment]
I guess this PR should close #1626.

**Comment by martinvw** [comment]
> @martinvw Not sure why you ask me.
&
> I guess this PR should close #1626.

Because of that. You had the (correct) opinion that should be improved so I asked your blessing :-)

**Comment by martinvw** [comment]
@mjagdis can you help me out with updating the documentation to include the changes you recently made. Can you give me some idea when you hope to be doing that? Thanks!

**Comment by mjagdis** [comment]
Try my mjagdis/rfxcom-doc branch :-)

**Comment by martinvw** [comment]
That looks great! I'll merge it in

**Comment by martinvw** [comment]
I decided to refer to the channel description instead of duplicating them, sorry for the extra work you did. If the users complain about that I will fix it :-)

You found a nice bug in my temperaturehumiditybarometric migration, it seems we have some unlinked channels. I'll fix it.

imho everything is ready.

If someone could do a final review? @metbril @Jamstah @paulianttila

**Comment by mjagdis** [comment]
It was all done by a perl script mounting the XML. I don't do "extra" work :-)

Although I think some of the XML could be a little better and I really wish the XML descriptions could be more than just strings.

**Comment by martinvw** [comment]
> Although I think some of the XML could be a little better and I really wish the XML descriptions could be more than just strings.

I don't know whether it is intended, but you can, see:

https://github.com/openhab/openhab2-addons/pull/2289/files#diff-3be6305afab6cd06c27b83e4613c69d6R10 

It was even more fancy, see https://github.com/openhab/openhab2-addons/pull/2289#issuecomment-307140132

**Comment by martinvw** [comment]
If someone could do a final review? @metbril @Jamstah @paulianttila please :-)

File: addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -155,18 +155,27 @@
...
 
-	<channel-type id="batterylevel" advanced="true">
-		<item-type>Number</item-type>
-		<label>Battery Level</label>
-		<description>Battery level</description>
-		<state min="0" max="9" step="1" readOnly="true"></state>
+	<channel-type id="forecast">
+		<item-type>String</item-type>
+		<label>Forecast</label>
+		<description>Forecast from device</description>
```
Comment (paulianttila) [inline addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml]: Should description be e.g. "Weather forecast"

Comment (martinvw) [inline addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml]: Yes

**Comment by martinvw** [comment]
@paulianttila thanks, I clarified the label was there anything else which you would like to have addressed?

**Pull request merged**